core/src/test/java/com/datastax/oss/driver/api/core/metadata/SerializableMetadataTest.java

```diff
@@ -0,0 +1,77 @@
+package com.datastax.oss.driver.api.core.metadata;
+
+import static org.junit.Assert.*;
+
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.metadata.schema.KeyspaceMetadata;
+import com.datastax.oss.driver.api.core.metadata.schema.TableMetadata;
+import com.datastax.oss.driver.internal.core.metadata.schema.DefaultKeyspaceMetadata;
+import com.datastax.oss.driver.internal.core.metadata.schema.DefaultTableMetadata;
+import org.apache.commons.lang3.SerializationUtils;
+import org.junit.Test;
+
+import java.net.InetSocketAddress;
+import java.util.Optional;
+
+public class SerializableMetadataTest {
+
+    /**
+     * Assuming the test below is run against the following:
```

Comment:
why do you mean assuming? I think that it should the schema (ks, tables) should be created in the `beforeAll` instead of the comment. (CREATE KEYSPACE IF NO EXISTS, CREATE TABLE IF NOT EXISTS)